.github/workflows/changelog-check.yml

```diff
@@ -0,0 +1,78 @@
+# **what?**
```

Comment:
I should have this #5501 done soon and then this can be updated to point there.  I would say this depends on that being done before it can me merged in.